nym-api/src/support/storage/manager.rs

```diff
@@ -553,7 +553,7 @@ impl StorageManager {
                 mixnode_result.identity,
                 mixnode_result.node_id,
             )
-            .fetch_one(&mut tx)
+            .fetch_one(&mut *tx)
```

Comment:
do those queries still work? it looks a bit dodgy to take reference to a dereferences transaction. I hope it doesn't end up copying it or something instead of reusing it

Reply:
> The Executor impls for Transaction and PoolConnection have been deleted because they cannot exist in the new crate architecture without rewriting the Executor trait entirely.
To fix this breakage, simply add a dereference where an impl Executor is expected, as they both dereference to the inner connection type which will still implement it:

mentioned [here](https://github.com/launchbadge/sqlx/blob/main/CHANGELOG.md#070---2023-06-30)